package.json

```diff
@@ -1,21 +1,35 @@
 {
-  "dependencies": {
-    "axios": "^0.18.0",
-    "colyseus": "^0.13.7",
-    "colyseus.js": "^0.13.0",
-    "express": "^4.16.3",
-    "pixi.js": "^4.8.1",
-    "random-color": "^1.0.1"
+  "name": "@tutturu-tv/rush",
+  "version": "1.0.0",
+  "description": "A fast game",
+  "main": "dist/src/index.js",
+  "scripts": {
+    "start": "node .",
+    "build": "bash ./scripts/build.sh",
```

Comment:
I decided to use one build script because we must compile TS before building the client part. I'm not sure that we can compile the client code separately from the server code because `client/defs.ts` uses server types, so a separate compilation will create a lot of extra code. Correct me if I am wrong.

Reply:
You'd be certainly doing the same thing... just point build:server to tsc, build:client to parcel, and do `yarn build:server && yarn build:client`

Reply:
Does it make sense? We will not be able to build the client without building the server, so it will confuse people.

Reply:
There are a few cases, like if you only need to rebuild the client but server is already compiled, etc.